vector/src/main/res/layout/fragment_location_live_map_view.xml

```diff
@@ -8,8 +8,9 @@
 
     <View
         android:id="@+id/liveLocationPopupAnchor"
-        android:layout_width="40dp"
-        android:layout_height="40dp" />
+        android:layout_width="0dp"
```

Comment:
why did you change this ? 
this is not related with the bug ?

Reply:
We use the bottom point of this anchor for the "share location" popup. And previously we were placing this anchor aligned with the top of the marker. So previously we needed to shift the anchor to the bottom as the amount of the height of the marker.

Now, we always show the popup at the center of the screen, not related to the position of the marker. If we continue using 40dp height then it will give a gap of 40dp between the pin and the popup.